tests/functional/login.js

```diff
@@ -0,0 +1,80 @@
+const Nightmare = require('nightmare');
+const assert = require('chai').assert;
+
+const url = 'http://localhost:3333/';
+
+
+
```

Comment:
Is there a reason for the extra new lines here?

Reply:
No, I changed it.